packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -326,6 +355,8 @@ class SemanticsNode extends AbstractNode {
   List<SemanticsNode> _newChildren;
 
   /// Append the given children as children of this node.
+  ///
+  /// You need to call [finalizeChildren] after all children have been added.
```

Comment:
Avoid the second person ("you") in documentation. We prefer the passive. "`The [finalizeChildren] method must be called after all children have been added, to ...`"